osbuild-composer.spec

```diff
@@ -334,6 +334,7 @@ Requires:   libvirt-daemon-driver-secret
 Requires:   libvirt-daemon-driver-storage
 Requires:   libvirt-daemon-driver-storage-disk
 Requires:   libvirt-daemon-kvm
+Requires:   lorax
```

Comment:
@teg Looks `mkksiso` not included in RHEL release. https://src.osci.redhat.com/rpms/lorax/blob/rhel-8.4.0/f/lorax.spec#_208. It's only available for Fedora.

Reply:
Yeah, this is a pain. I got this working locally by spinning up a container, but that might be overkill.

Reply:
Is it in EPEL? If so, we can just add it here: https://github.com/osbuild/osbuild-composer/blob/c2dc9d65fb71add09d5afc4f6e66991c46a7a1c0/tools/provision.sh#L11

Reply:
@bcl is there a convenient way to get `mkksiso` on RHEL8? Is it in EPEL?